src/shogun/evaluation/EvaluationResult.h

```diff
@@ -20,7 +20,9 @@ namespace shogun
 enum EEvaluationResultType
 {
 	CROSSVALIDATION_RESULT=0,
-	GRADIENTEVALUATION_RESULT=1
+	GRADIENTEVALUATION_RESULT=1,
+	CROSSVALIDATIONSTORAGE_RESULT=2,
+	CROSSVALIDATIONFOLDSTORAGE_RESULT=3
```

Comment:
is this used anywhere? I thought we dont need to dispatch the type since we use "get"?

Reply:
We still have some `obtain_from_generic` which use those enums. Maybe they can be removed though

Reply:
yeah just remove that